packages/twenty-front/src/modules/billing/components/SettingsBillingMonthlyCreditsSection.tsx

```diff
@@ -41,46 +51,61 @@ export const SettingsBillingMonthlyCreditsSection = () => {
     : formatNumber(freeUsageQuantity);
 
   return (
-    <Section>
-      <H2Title
-        title={t`Monthly Credits`}
-        description={t`Track your monthly workflow credit consumption.`}
-      />
-      <SubscriptionInfoContainer>
-        <SettingsBillingLabelValueItem
-          label={t`Free Credits Used`}
-          value={`${formattedFreeUsageQuantity}/${formatAmount(includedFreeQuantity)}`}
-        />
-        <ProgressBar
-          value={progressBarValue}
-          barColor={
-            isFreeCreditProgressBarCompleted
-              ? BACKGROUND_LIGHT.quaternary
-              : COLOR.blue
-          }
-          backgroundColor={BACKGROUND_LIGHT.tertiary}
-          withBorderRadius={true}
+    <>
+      <Section>
+        <H2Title
+          title={t`Monthly Credits`}
+          description={t`Track your monthly workflow credit consumption.`}
         />
-
-        <StyledLineSeparator />
-        {!isTrialing && (
+        <SubscriptionInfoContainer>
           <SettingsBillingLabelValueItem
-            label={t`Extra Credits Used`}
-            value={`${formatNumber(paidUsageQuantity)}`}
+            label={t`Free Credits Used`}
```

Comment:
Are those really free credits? Or it's Plan Credits? Like how much credits from your selected plan have you used?